about.html

```diff
@@ -0,0 +1,115 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Baloo+Da+2:wght@400;500;600;700;800&family=Josefin+Slab:ital,wght@0,300;0,400;0,500;1,300;1,400&family=Lato:ital,wght@0,300;1,300;1,400&family=Mulish:ital,wght@0,300;0,400;0,500;0,600;1,200;1,300;1,400;1,500&family=Poppins:ital,wght@0,200;0,300;1,100;1,200;1,300&display=swap" rel="stylesheet">
+        <link rel="stylesheet" href="about.css">
+    <title>About Page</title>
+</head>
+<body>
+  <header>
+    <div class="container"  id="About">
+		<nav id="socila-contact" class="socila-contact">
+			<a href="#socila-contact">
+				<img class="facebook" src="imgs/facebook.svg" alt="facebook">
+			</a>
+			<a href="#socila-contact">
+				<img class="twitter" src="imgs/twitter.svg" alt="twitter">
+			</a>
+			<a href="#">English</a>
+			<a href="#">My Page</a>
+			<a href="#">Logout</a>
+		 </nav>
+		 <nav class="desktop-nav">
+			<div class="left-desk"> 
+			<a href="#">
+				 <img class="desktop-logo" src="imgs/logo.png" alt="">
+			 </a>
+			</div>
+			<div class="right-desk">
+			 <a class="links" href="#">About</a>
+			 <a class="links" href="#">Program</a>
+			 <a class="links" href="#">Join</a>
+			 <a class="links" href="#">Partners</a>
+			 <a class="links" href="#">News</a>
+			 <div class="desktop-nav-btn">
+				 <div class="desktop-btn">ZTM Course</div>
+			 </div>
+			</div>
+		 </nav>
```

Comment:
- [ ] Kindly make users can navigate between the home page and the about page🙄. I can't navigate to the home page from the about page😌, I'd suggest you fix that for a better user experience😎

Reply:
- [ ] Please address this issue from the previous reviewer. I still cannot navigate back to the home page.

Reply:
✔ Sorry I was looking at an outdated version of your project. this is completed your project should be approved.